projects/packages/jetpack-mu-wpcom/tests/php/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad-test.php

```diff
@@ -89,7 +89,7 @@ public function test_update_checklist_statuses() {
 
 		$values = array(
 			'domain_upsell_deferred' => true,
-			'site_launched'          => true,
+			'site_launched'          => false,
```

Comment:
Could you explain why you made this change?

Reply:
Since I changed the code that was being used to store this values I thought changing this was a good opportunity to exercise both the true and false code paths.

Reply:
:+1: